logging/kit/shared_test.go

```diff
@@ -0,0 +1,93 @@
+package kit_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/grpc-ecosystem/go-grpc-middleware/logging/kit/ctxkit"
+	"io"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	grpc_ctxtags "github.com/grpc-ecosystem/go-grpc-middleware/tags"
+	grpc_testing "github.com/grpc-ecosystem/go-grpc-middleware/testing"
+	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/testing/testproto"
+	"golang.org/x/net/context"
```

Comment:
+1

Reply:
Also can you make sure you run goimports over the files :) might be something you want to set up as a filewatcher in the IDE if you can